Gds/src/fprime_gds/executables/run_deployment.py

```diff
@@ -295,7 +296,7 @@ def launch_app(app, port, address, logs, **_):
     :param app: application to launch
     :param port: port to connect to
     :param address: address to connect to
-    :param log_dir: log directory to place files into
+    :param logs: TODO: the parameter description should be filled by maintainer
```

Comment:
logs most likely replaced log_dir. The same description can be used here:
`log directory to place files into`